pkg/apis/compliance/v1alpha1/compliancecheckresult_types.go

```diff
@@ -85,7 +86,8 @@ type ComplianceCheckResult struct {
 	// The severity of a check status
 	Severity ComplianceCheckResultSeverity `json:"severity"`
 	// A human-readable check description, what and why it does
-	Description string `json:"description,omitempty"`
+	ValueUsed   []string `json:"valuesUsed,omitempty"`
```

Comment:
Given that this is a list, I suggest using `ValuesUsed`, denoting the possibility of several values.